In [8]:
!wget -O data.sqlite https://raw.githubusercontent.com/Handit14/OrbitFutureAcademy_KampusMerdeka/main/Materi/SQL/data.sqlite

--2021-11-13 15:39:59--  https://raw.githubusercontent.com/Handit14/OrbitFutureAcademy_KampusMerdeka/main/Materi/SQL/data.sqlite
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.111.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 16384 (16K) [application/octet-stream]
Saving to: ‘data.sqlite’

data.sqlite         100%[===================>]  16.00K  --.-KB/s    in 0s      

2021-11-13 15:40:00 (102 MB/s) - ‘data.sqlite’ saved [16384/16384]



In [9]:
import sqlite3
import pandas as pd

# Connect sqlite3 to python

In [10]:
connection = sqlite3.connect('data.sqlite')


# Operasi Insert

In [11]:
result = connection.cursor()
command = "INSERT INTO AGENTS VALUES ('A013', 'Achmad Malik', 'Indonesia', '0.27', '021-14045', '')"
result.execute(command)

# Operasi Select

In [13]:
result = connection.cursor()
command = "select * from agents"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

AGENT_CODE    AGENT_NAME WORKING_AREA COMMISSION      PHONE_NO COUNTRY
0        A001      Subbarao    Bangalore       0.14  077-12346674        
1        A002        Mukesh       Mumbai       0.11  029-12358964        
2        A003         Alex        London       0.13  075-12458969        
3        A004          Ivan      Torento       0.15  008-22544166        
4        A005      Anderson      Brisban       0.13  045-21447739        
5        A006         McDen       London       0.15  078-22255588        
6        A007    Ramasundar    Bangalore       0.15  077-25814763        
7        A008        Alford     New York       0.12  044-25874365        
8        A009      Benjamin    Hampshair       0.11  008-22536178        
9        A010    Santakumar      Chennai       0.14  007-22388644        
10       A011    Ravi Kumar    Bangalore       0.15  077-45625874        
11       A012        Lucida     San Jose       0.12  044-52981425        
12       A013  Achmad Malik    Indonesia       0.27     021-14045

In [14]:
result = connection.cursor()
command = "select * from orders"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

ORD_NUM ORD_AMOUNT ADVANCE_AMOUNT  ... CUST_CODE AGENT_CODE ORD_DESCRIPTION
0   200100    1000.00         600.00  ...    C00013       A003             SOD
1   200101    3000.00        1000.00  ...    C00001       A008             SOD
2   200102    2000.00         300.00  ...    C00012       A012             SOD
3   200103    1500.00         700.00  ...    C00021       A005             SOD
4   200104    1500.00         500.00  ...    C00006       A004             SOD
5   200105    2500.00         500.00  ...    C00025       A011             SOD
6   200106    2500.00         700.00  ...    C00005       A002             SOD
7   200107    4500.00         900.00  ...    C00007       A010             SOD
8   200108    4000.00         600.00  ...    C00008       A004             SOD
9   200109    3500.00         800.00  ...    C00011       A010             SOD
10  200110    3000.00         500.00  ...    C00019       A010             SOD
11  200111    1000.00         300.00  ...    C00020       A008             SOD
12  200112    2000.00         400.00  ...    C00016       A007             SOD
13  200113    4000.00         600.00  ...    C00022       A002             SOD
14  200114    3500.00        2000.00  ...    C00002       A008             SOD
15  200116     500.00         100.00  ...    C00010       A009             SOD
16  200117     800.00         200.00  ...    C00014       A001             SOD
17  200118     500.00         100.00  ...    C00023       A006             SOD
18  200119    4000.00         700.00  ...    C00007       A010             SOD
19  200120     500.00         100.00  ...    C00009       A002             SOD
20  200121    1500.00         600.00  ...    C00008       A004             SOD
21  200122    2500.00         400.00  ...    C00003       A004             SOD
22  200123     500.00         100.00  ...    C00022       A002             SOD
23  200124     500.00         100.00  ...    C00017       A007             SOD
24  200125    2000.00         600.00  ...    C00018       A005             SOD
25  200126     500.00         100.00  ...    C00022       A002             SOD
26  200127    2500.00         400.00  ...    C00015       A003             SOD
27  200128    3500.00        1500.00  ...    C00009       A002             SOD
28  200129    2500.00         500.00  ...    C00024       A006             SOD
29  200130    2500.00         400.00  ...    C00025       A011             SOD
30  200131     900.00         150.00  ...    C00012       A012             SOD
31  200133    1200.00         400.00  ...    C00009       A002             SOD
32  200134    4200.00        1800.00  ...    C00004       A005             SOD
33  200135    2000.00         800.00  ...    C00007       A010             SOD

[34 rows x 7 columns]

In [15]:
result = connection.cursor()
command = "select * from customer"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_CODE    CUST_NAME  CUST_CITY  ... OUTSTANDING_AMT      PHONE_NO AGENT_CODE
0     C00001      Micheal   New York  ...         6000.00       CCCCCCC       A008
1     C00002         Bolt   New York  ...         3000.00       DDNRDRH       A008
2     C00003       Martin    Torento  ...         8000.00       MJYURFD       A004
3     C00004      Winston    Brisban  ...         6000.00       AAAAAAA       A005
4     C00005     Sasikant     Mumbai  ...        11000.00  147-25896312       A002
5     C00006      Shilton    Torento  ...        11000.00       DDDDDDD       A004
6     C00007   Ramanathan    Chennai  ...         9000.00       GHRDWSD       A010
7     C00008     Karolina    Torento  ...         5000.00       HJKORED       A004
8     C00009       Ramesh     Mumbai  ...        12000.00      Phone No       A002
9     C00010      Charles  Hampshair  ...         5000.00       MMMMMMM       A009
10    C00011    Sundariya    Chennai  ...        11000.00       PPHGRTS       A010
11    C00012       Steven   San Jose  ...         3000.00       KRFYGJK       A012
12    C00013       Holmes     London  ...         4000.00       BBBBBBB       A003
13    C00014   Rangarappa  Bangalore  ...        12000.00       AAAATGF       A001
14    C00015       Stuart     London  ...        11000.00       GFSGERS       A003
15    C00016   Venkatpati  Bangalore  ...        12000.00       JRTVFDD       A007
16    C00017     Srinivas  Bangalore  ...         9000.00       AAAAAAB       A007
17    C00018      Fleming    Brisban  ...         5000.00       NHBGVFC       A005
18    C00019  Yearannaidu    Chennai  ...         8000.00       ZZZZBFV       A010
19    C00020       Albert   New York  ...         6000.00       BBBBSBB       A008
20    C00021        Jacks    Brisban  ...         7000.00       WERTGDF       A005
21    C00022      Avinash     Mumbai  ...         9000.00  113-12345678       A002
22    C00023         Karl     London  ...         3000.00       AAAABAA       A006
23    C00024         Cook     London  ...         6000.00       FSDDSDF       A006
24    C00025    Ravindran  Bangalore  ...         8000.00       AVAVAVA       A011

[25 rows x 12 columns]

## Contoh : menampilkan sebagian/beberapa kolom pada  tabel Customer.

In [16]:
result = connection.cursor()
command = "SELECT CUST_NAME,CUST_CITY AS 'Kota Asal' FROM customer"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_NAME  Kota Asal
0       Micheal   New York
1          Bolt   New York
2        Martin    Torento
3       Winston    Brisban
4      Sasikant     Mumbai
5       Shilton    Torento
6    Ramanathan    Chennai
7      Karolina    Torento
8        Ramesh     Mumbai
9       Charles  Hampshair
10    Sundariya    Chennai
11       Steven   San Jose
12       Holmes     London
13   Rangarappa  Bangalore
14       Stuart     London
15   Venkatpati  Bangalore
16     Srinivas  Bangalore
17      Fleming    Brisban
18  Yearannaidu    Chennai
19       Albert   New York
20        Jacks    Brisban
21      Avinash     Mumbai
22         Karl     London
23         Cook     London
24    Ravindran  Bangalore

# Operasi Update

In [17]:
result = connection.cursor()
command = "SELECT CUST_NAME,GRADE FROM customer"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_NAME GRADE
0       Micheal     2
1          Bolt     3
2        Martin     2
3       Winston     1
4      Sasikant     1
5       Shilton     1
6    Ramanathan     1
7      Karolina     1
8        Ramesh     3
9       Charles     3
10    Sundariya     3
11       Steven     1
12       Holmes     2
13   Rangarappa     2
14       Stuart     1
15   Venkatpati     2
16     Srinivas     2
17      Fleming     2
18  Yearannaidu     1
19       Albert     3
20        Jacks     1
21      Avinash     2
22         Karl     0
23         Cook     2
24    Ravindran     2

In [18]:
result = connection.cursor()
command = "UPDATE customer SET GRADE = 1 WHERE CUST_NAME = 'Micheal'"
result.execute(command)


In [19]:
result = connection.cursor()
command = "SELECT CUST_NAME,GRADE FROM customer"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_NAME GRADE
0       Micheal     1
1          Bolt     3
2        Martin     2
3       Winston     1
4      Sasikant     1
5       Shilton     1
6    Ramanathan     1
7      Karolina     1
8        Ramesh     3
9       Charles     3
10    Sundariya     3
11       Steven     1
12       Holmes     2
13   Rangarappa     2
14       Stuart     1
15   Venkatpati     2
16     Srinivas     2
17      Fleming     2
18  Yearannaidu     1
19       Albert     3
20        Jacks     1
21      Avinash     2
22         Karl     0
23         Cook     2
24    Ravindran     2

# Operasi Delete

In [20]:
result = connection.cursor()
command = "DELETE FROM customer WHERE CUST_NAME = 'Micheal'"
result.execute(command)

In [21]:
result = connection.cursor()
command = "SELECT * FROM customer"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_CODE    CUST_NAME  CUST_CITY  ... OUTSTANDING_AMT      PHONE_NO AGENT_CODE
0     C00002         Bolt   New York  ...         3000.00       DDNRDRH       A008
1     C00003       Martin    Torento  ...         8000.00       MJYURFD       A004
2     C00004      Winston    Brisban  ...         6000.00       AAAAAAA       A005
3     C00005     Sasikant     Mumbai  ...        11000.00  147-25896312       A002
4     C00006      Shilton    Torento  ...        11000.00       DDDDDDD       A004
5     C00007   Ramanathan    Chennai  ...         9000.00       GHRDWSD       A010
6     C00008     Karolina    Torento  ...         5000.00       HJKORED       A004
7     C00009       Ramesh     Mumbai  ...        12000.00      Phone No       A002
8     C00010      Charles  Hampshair  ...         5000.00       MMMMMMM       A009
9     C00011    Sundariya    Chennai  ...        11000.00       PPHGRTS       A010
10    C00012       Steven   San Jose  ...         3000.00       KRFYGJK       A012
11    C00013       Holmes     London  ...         4000.00       BBBBBBB       A003
12    C00014   Rangarappa  Bangalore  ...        12000.00       AAAATGF       A001
13    C00015       Stuart     London  ...        11000.00       GFSGERS       A003
14    C00016   Venkatpati  Bangalore  ...        12000.00       JRTVFDD       A007
15    C00017     Srinivas  Bangalore  ...         9000.00       AAAAAAB       A007
16    C00018      Fleming    Brisban  ...         5000.00       NHBGVFC       A005
17    C00019  Yearannaidu    Chennai  ...         8000.00       ZZZZBFV       A010
18    C00020       Albert   New York  ...         6000.00       BBBBSBB       A008
19    C00021        Jacks    Brisban  ...         7000.00       WERTGDF       A005
20    C00022      Avinash     Mumbai  ...         9000.00  113-12345678       A002
21    C00023         Karl     London  ...         3000.00       AAAABAA       A006
22    C00024         Cook     London  ...         6000.00       FSDDSDF       A006
23    C00025    Ravindran  Bangalore  ...         8000.00       AVAVAVA       A011

[24 rows x 12 columns]

# SELECT dengan Kriteria : Operator Perbandingan


In [22]:
result = connection.cursor()
command = "SELECT * FROM agents WHERE AGENT_CODE='A002';"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

AGENT_CODE AGENT_NAME WORKING_AREA COMMISSION      PHONE_NO COUNTRY
0       A002     Mukesh       Mumbai       0.11  029-12358964

# SELECT dengan Kriteria : Operator Boolean


In [23]:
result = connection.cursor()
command = "SELECT * FROM orders WHERE ORD_AMOUNT >= 500 AND ADVANCE_AMOUNT >= 300;"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

ORD_NUM ORD_AMOUNT ADVANCE_AMOUNT  ... CUST_CODE AGENT_CODE ORD_DESCRIPTION
0   200100    1000.00         600.00  ...    C00013       A003             SOD
1   200101    3000.00        1000.00  ...    C00001       A008             SOD
2   200102    2000.00         300.00  ...    C00012       A012             SOD
3   200103    1500.00         700.00  ...    C00021       A005             SOD
4   200104    1500.00         500.00  ...    C00006       A004             SOD
5   200105    2500.00         500.00  ...    C00025       A011             SOD
6   200106    2500.00         700.00  ...    C00005       A002             SOD
7   200107    4500.00         900.00  ...    C00007       A010             SOD
8   200108    4000.00         600.00  ...    C00008       A004             SOD
9   200109    3500.00         800.00  ...    C00011       A010             SOD
10  200110    3000.00         500.00  ...    C00019       A010             SOD
11  200111    1000.00         300.00  ...    C00020       A008             SOD
12  200112    2000.00         400.00  ...    C00016       A007             SOD
13  200113    4000.00         600.00  ...    C00022       A002             SOD
14  200114    3500.00        2000.00  ...    C00002       A008             SOD
15  200116     500.00         100.00  ...    C00010       A009             SOD
16  200117     800.00         200.00  ...    C00014       A001             SOD
17  200118     500.00         100.00  ...    C00023       A006             SOD
18  200119    4000.00         700.00  ...    C00007       A010             SOD
19  200120     500.00         100.00  ...    C00009       A002             SOD
20  200121    1500.00         600.00  ...    C00008       A004             SOD
21  200122    2500.00         400.00  ...    C00003       A004             SOD
22  200123     500.00         100.00  ...    C00022       A002             SOD
23  200124     500.00         100.00  ...    C00017       A007             SOD
24  200125    2000.00         600.00  ...    C00018       A005             SOD
25  200126     500.00         100.00  ...    C00022       A002             SOD
26  200127    2500.00         400.00  ...    C00015       A003             SOD
27  200128    3500.00        1500.00  ...    C00009       A002             SOD
28  200129    2500.00         500.00  ...    C00024       A006             SOD
29  200130    2500.00         400.00  ...    C00025       A011             SOD
30  200131     900.00         150.00  ...    C00012       A012             SOD
31  200133    1200.00         400.00  ...    C00009       A002             SOD
32  200134    4200.00        1800.00  ...    C00004       A005             SOD
33  200135    2000.00         800.00  ...    C00007       A010             SOD

[34 rows x 7 columns]

# SELECT dengan Kriteria : Operator LIKE

In [24]:
result = connection.cursor()
command = "SELECT * FROM customer WHERE CUST_NAME LIKE '%S%'"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data


CUST_CODE  CUST_NAME  CUST_CITY  ... OUTSTANDING_AMT      PHONE_NO AGENT_CODE
0     C00004    Winston    Brisban  ...         6000.00       AAAAAAA       A005
1     C00005   Sasikant     Mumbai  ...        11000.00  147-25896312       A002
2     C00006    Shilton    Torento  ...        11000.00       DDDDDDD       A004
3     C00009     Ramesh     Mumbai  ...        12000.00      Phone No       A002
4     C00010    Charles  Hampshair  ...         5000.00       MMMMMMM       A009
5     C00011  Sundariya    Chennai  ...        11000.00       PPHGRTS       A010
6     C00012     Steven   San Jose  ...         3000.00       KRFYGJK       A012
7     C00013     Holmes     London  ...         4000.00       BBBBBBB       A003
8     C00015     Stuart     London  ...        11000.00       GFSGERS       A003
9     C00017   Srinivas  Bangalore  ...         9000.00       AAAAAAB       A007
10    C00021      Jacks    Brisban  ...         7000.00       WERTGDF       A005
11    C00022    Avinash     Mumbai  ...         9000.00  113-12345678       A002

[12 rows x 12 columns]

# SELECT dengan Kriteria : Operator IN dan NOT IN


In [25]:
result = connection.cursor()
command = "SELECT * FROM customer WHERE CUST_COUNTRY IN ('USA','Canada','India');"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_CODE    CUST_NAME  CUST_CITY  ... OUTSTANDING_AMT      PHONE_NO AGENT_CODE
0     C00002         Bolt   New York  ...         3000.00       DDNRDRH       A008
1     C00003       Martin    Torento  ...         8000.00       MJYURFD       A004
2     C00005     Sasikant     Mumbai  ...        11000.00  147-25896312       A002
3     C00006      Shilton    Torento  ...        11000.00       DDDDDDD       A004
4     C00007   Ramanathan    Chennai  ...         9000.00       GHRDWSD       A010
5     C00008     Karolina    Torento  ...         5000.00       HJKORED       A004
6     C00009       Ramesh     Mumbai  ...        12000.00      Phone No       A002
7     C00011    Sundariya    Chennai  ...        11000.00       PPHGRTS       A010
8     C00012       Steven   San Jose  ...         3000.00       KRFYGJK       A012
9     C00014   Rangarappa  Bangalore  ...        12000.00       AAAATGF       A001
10    C00016   Venkatpati  Bangalore  ...        12000.00       JRTVFDD       A007
11    C00017     Srinivas  Bangalore  ...         9000.00       AAAAAAB       A007
12    C00019  Yearannaidu    Chennai  ...         8000.00       ZZZZBFV       A010
13    C00020       Albert   New York  ...         6000.00       BBBBSBB       A008
14    C00022      Avinash     Mumbai  ...         9000.00  113-12345678       A002
15    C00025    Ravindran  Bangalore  ...         8000.00       AVAVAVA       A011

[16 rows x 12 columns]

In [26]:
result = connection.cursor()
command = "SELECT * FROM customer WHERE CUST_COUNTRY NOT IN ('USA','Canada','India');"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_CODE CUST_NAME  CUST_CITY  ... OUTSTANDING_AMT PHONE_NO AGENT_CODE
0    C00004   Winston    Brisban  ...         6000.00  AAAAAAA       A005
1    C00010   Charles  Hampshair  ...         5000.00  MMMMMMM       A009
2    C00013    Holmes     London  ...         4000.00  BBBBBBB       A003
3    C00015    Stuart     London  ...        11000.00  GFSGERS       A003
4    C00018   Fleming    Brisban  ...         5000.00  NHBGVFC       A005
5    C00021     Jacks    Brisban  ...         7000.00  WERTGDF       A005
6    C00023      Karl     London  ...         3000.00  AAAABAA       A006
7    C00024      Cook     London  ...         6000.00  FSDDSDF       A006

[8 rows x 12 columns]

# SELECT dengan Kriteria : Operator BETWEEN


In [27]:
result = connection.cursor()
command = "SELECT * FROM orders WHERE ADVANCE_AMOUNT BETWEEN '300' AND '800'"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

ORD_NUM ORD_AMOUNT ADVANCE_AMOUNT  ... CUST_CODE AGENT_CODE ORD_DESCRIPTION
0   200100    1000.00         600.00  ...    C00013       A003             SOD
1   200102    2000.00         300.00  ...    C00012       A012             SOD
2   200103    1500.00         700.00  ...    C00021       A005             SOD
3   200104    1500.00         500.00  ...    C00006       A004             SOD
4   200105    2500.00         500.00  ...    C00025       A011             SOD
5   200106    2500.00         700.00  ...    C00005       A002             SOD
6   200108    4000.00         600.00  ...    C00008       A004             SOD
7   200110    3000.00         500.00  ...    C00019       A010             SOD
8   200111    1000.00         300.00  ...    C00020       A008             SOD
9   200112    2000.00         400.00  ...    C00016       A007             SOD
10  200113    4000.00         600.00  ...    C00022       A002             SOD
11  200119    4000.00         700.00  ...    C00007       A010             SOD
12  200121    1500.00         600.00  ...    C00008       A004             SOD
13  200122    2500.00         400.00  ...    C00003       A004             SOD
14  200125    2000.00         600.00  ...    C00018       A005             SOD
15  200127    2500.00         400.00  ...    C00015       A003             SOD
16  200129    2500.00         500.00  ...    C00024       A006             SOD
17  200130    2500.00         400.00  ...    C00025       A011             SOD
18  200133    1200.00         400.00  ...    C00009       A002             SOD

[19 rows x 7 columns]

# Mengurutkan Data


In [28]:
result = connection.cursor()
command = "SELECT * FROM customer ORDER BY CUST_NAME ASC;"
result.execute(command)
results = result.fetchall()
cols = [column[0] for column in result.description]
results_data= pd.DataFrame.from_records(results, columns = cols)

results_data

CUST_CODE    CUST_NAME  CUST_CITY  ... OUTSTANDING_AMT      PHONE_NO AGENT_CODE
0     C00020       Albert   New York  ...         6000.00       BBBBSBB       A008
1     C00022      Avinash     Mumbai  ...         9000.00  113-12345678       A002
2     C00002         Bolt   New York  ...         3000.00       DDNRDRH       A008
3     C00010      Charles  Hampshair  ...         5000.00       MMMMMMM       A009
4     C00024         Cook     London  ...         6000.00       FSDDSDF       A006
5     C00018      Fleming    Brisban  ...         5000.00       NHBGVFC       A005
6     C00013       Holmes     London  ...         4000.00       BBBBBBB       A003
7     C00021        Jacks    Brisban  ...         7000.00       WERTGDF       A005
8     C00023         Karl     London  ...         3000.00       AAAABAA       A006
9     C00008     Karolina    Torento  ...         5000.00       HJKORED       A004
10    C00003       Martin    Torento  ...         8000.00       MJYURFD       A004
11    C00007   Ramanathan    Chennai  ...         9000.00       GHRDWSD       A010
12    C00009       Ramesh     Mumbai  ...        12000.00      Phone No       A002
13    C00014   Rangarappa  Bangalore  ...        12000.00       AAAATGF       A001
14    C00025    Ravindran  Bangalore  ...         8000.00       AVAVAVA       A011
15    C00005     Sasikant     Mumbai  ...        11000.00  147-25896312       A002
16    C00006      Shilton    Torento  ...        11000.00       DDDDDDD       A004
17    C00017     Srinivas  Bangalore  ...         9000.00       AAAAAAB       A007
18    C00012       Steven   San Jose  ...         3000.00       KRFYGJK       A012
19    C00015       Stuart     London  ...        11000.00       GFSGERS       A003
20    C00011    Sundariya    Chennai  ...        11000.00       PPHGRTS       A010
21    C00016   Venkatpati  Bangalore  ...        12000.00       JRTVFDD       A007
22    C00004      Winston    Brisban  ...         6000.00       AAAAAAA       A005
23    C00019  Yearannaidu    Chennai  ...         8000.00       ZZZZBFV       A010

[24 rows x 12 columns]

# Acitivty

Jawablah pertanyaan-pertanyaan berikut dengan SQL.
1. Tampilkan data customer yg mempunyai opening amount diatas 7000.

2. Tampilkan informasi order yg dilakukan melalui agensi dengan code A003.

3. Tampilkan data customer yg tinggal di New York, Bangalore, Mumbai, dan Torento.

4. Tampilkan informasi customer yang mempunyai payment amount diatas 7000 dan urutkan berdasarkan opening amountnya dari yg terbesar

5. Tampilkan data agensi yang tidak bekerja di london dan memiliki comission lebih dari 0.12
